webapp/app/protected/users/user/template.hbs

```diff
@@ -1,77 +1,79 @@
 <div class="section">
-	<h4>
-		{{#if model.isAdmin }}
```

Comment:
You probably want to check the current connected user `session.user.isAdmin`. Not the displayed user